app/src/main/java/com/nutomic/syncthingandroid/service/RestApi.java

```diff
@@ -494,6 +476,14 @@ private void normalizeDeviceId(String id, OnResultListener1<String> listener,
         });
     }
 
+
+    /**
+     * Updates cached folder and device completion info according to event data.
+     */
+    public void updateCompletionInfo(String deviceId, String folderId, CompletionInfo completionInfo) {
+        mCompletion.setCompletionInfo(deviceId, folderId, completionInfo);
+    }
```

Comment:
This function literally does nothing.

Reply:
It's a setter to avoid exposing the whole mCompletion instance to all mRestApi parents.

Reply:
Why not call it setCompletionInfo then?

Reply:
ok